src/BuiltInTools/dotnet-watch/Internal/FileWatcher/ChangeKind.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.DotNet.Watcher.Internal;
+
+internal enum ChangeKind
+{
+    Update,
```

Comment:
I sometimes see code that is sensitive to "Move" as being different from "Update".  Is there a case for that here?

Reply:
Not sure what you mean. I'm guessing you're suggesting adding "Move" ChangeKind.
I think that might need to be done to address https://github.com/dotnet/sdk/issues/43320.